react-front-end/tsrc/components/UserSearch.tsx

```diff
@@ -101,6 +104,7 @@ const UserSearch = ({
     OEQ.Common.UuidString | undefined
   >(undefined);
   const [hasSearched, setHasSearched] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<String | null>(null);
```

Comment:
Typically we just use `undefined`, so:

```suggestion
  const [errorMessage, setErrorMessage] = useState<String>();
```

And then update the usages accordingly.